### PR TITLE
New StatsWidget API for examples

### PR DESCRIPTION
**Pull request by tsherif**
- Bump probe.gl version
- Update examples to use latest StatsWidget API
- Some eslint fixes

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "camelcase": 0,
     "tree-shaking/no-side-effects-in-initialization": 0,
     "max-statements": 0,
-    "luma-gl-custom-rules/check-log-call": 1
+    "luma-gl-custom-rules/check-log-call": 1,
+    "spaced-comment": ["error", "always", {"exceptions": ["/"]}]
```
Comment (ibgreen) [inline .eslintrc]: Why would we diverge from this rule, all our code in all our repos follows it as does all other applications we work with? 

Making a local eslint-disable that is one thing, but changing it globally should really be something we agree on doing across our repos/in ocular-dev-tools.

File: examples/core/3d-texture/app.js [hunk 2]
```diff
@@ -133,27 +133,34 @@ class AppAnimationLoop extends AnimationLoop {
...
+        'GPU Time': 'averageTime',
+        'Frame Rate': 'fps'
+      },
+      resetOnUpdate: {
+        'CPU Time': true,
+        'GPU Time': true,
+        'Frame Rate': true
+      }
     });
-    statsWidget.setFormatter('CPU Time', stat => `CPU Time: ${stat.getAverageTime().toFixed(2)}ms`);
```
Comment (ibgreen) [inline examples/core/3d-texture/app.js]: Thanks!!!

File: examples/core/3d-texture/app.js [hunk 3]
```diff
@@ -133,27 +133,34 @@ class AppAnimationLoop extends AnimationLoop {
...
     return {cloud, mvpMat, viewMat, statsWidget};
   }
 
   onRender(animationProps) {
     const {gl, cloud, mvpMat, viewMat, statsWidget, tick, aspect} = animationProps;
+    statsWidget.update();
 
     mvpMat.perspective({fov: radians(75), aspect, near: NEAR, far: FAR}).multiplyRight(viewMat);
 
-    if (tick % 60 === 10) {
```
Comment (ibgreen) [inline examples/core/3d-texture/app.js]: +1

File: .eslintrc (hunk 1 above)
Comment (tsherif) [reply .eslintrc]: It's still on for almost all cases, but the '/' exception allows for this kind of header comment: https://github.com/uber/luma.gl/blob/master/examples/core/dof/app.js#L244

I can make that local to the file, though, until we discuss it further.

**Comment by tsherif** [comment]
Ok, eslint restored.

> Ideally we should run a "prettier" pass separately as the diff becomes quite big with all the styling updates and harder to see the actual changes.

This was kind of weird. It seems the dof example wasn't being linted before, and I guess some other update included it in the linting. So had to go in and fix everything so I could commit.

File: .eslintrc (hunk 1 above)
Comment (ibgreen) [reply .eslintrc]: Oh this style? 
```
///////////////////////////////////////
//
///////////////////////////////////////
```

That doesn't seem unreasonable, if you really want it widely used. I don't think you will get any major pushback on that, as long as the rule doesn't open any "other doors".

Worth remarking that our linter definitions are old, we should probably upgrade to a modern widely used standard. Uber has a new set of definitions that we should consider (we are using an older version).

We just need someone who cares enough about it to make a proposal.

**Pull request merged**